README.md

```diff
@@ -1,30 +1,163 @@
+    1. Download Yarn Package manager [here](https://classic.yarnpkg.com/en/docs/install)
+    2. If you already have `npm` installed you can simply type in `npm install --global yarn`.
+    3. If you already have `choco` installed you can simply type in `choco install yarn` 
+    4. If you already have `scoop` installed you can simply type in `scoop install yarn` 
+3. To check if the yarn has installed correctly type in `yarn --version` this should show you the current version of yarn.
+
+### 🚩 Running in Deployment mode :
+
+1. Open the terminal and type in `yarn start`
+2. Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+3. The page will reload if you make edits.
+4. You will also see any lint errors in the console.
+
+### 🚩 Testing changes :
+
+1. After changes type `yarn test`. This launches the test runner in the interactive watch mode.
+2. See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information. 
+
+### 🚩 Building project :
+1. Run the command `yarn build`
+2. Builds the app for production to the `build` folder.\
+3. It correctly bundles React in production mode and optimizes the build for the best performance.
+4. The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
+5. See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## 📌 Pushing your changes
+
+Now you have made the changes , tested them, and built them. so now its time to push them.
+1. Goto your terminal and type `git status` and hit enter, this will show your changes from the files
+2. Then type in `git add` and hit enter, this will add all the files to staging area
+3. Commit the changes by `git commit -m "<message-describing-your-change>"` and hit enter.
+4. Now push your branch to your fork by `git push origin <your-branch-name>` and hit enter.
+
+## 📌 Creating a pull request
+
+By this time you can see a message on your github fork as `your fork is ahead of codeforcause:master by <number> of commits` and also you can see a button called `Compare and pull request`. 
+1. Click on `Compare and pull request` button.(Refer image no 1)
+2. You will see a template.(Refer image no 2)
+3. Fill out the template completely by describing your change, cause of change, issue getting fixed etc.
+4. After filling the template completely click on `Pull request`
+<img src="./doc_imgaes/compareAndpull.png" alt="Demo copy image" align="center"/>
+<img src="./doc_imgaes/pull.png" alt="Demo copy image" align="center"/>
+
+## 📌 Guiders / Maintainers :
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[Abhishek Kumar](https://github.com/Abhishek-kumar09)\
+[Kunal Kushwaha](https://github.com/kunal-kushwaha)\
+[Siddhant Khare](https://github.com/Siddhant-K-code)\
```

Comment:
I am not the official Maintainer as of now, hope @KeenWarrior will make me soon, till then remove my name from there

Reply:
ok

Reply:
done